nexus/db-model/src/utilization.rs

```diff
@@ -57,39 +57,15 @@ impl From<SiloUtilization> for views::Utilization {
     }
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
-pub struct Ipv4Utilization {
-    pub allocated: u32,
-    pub capacity: u32,
-}
-
-#[derive(Debug, Clone, Serialize, Deserialize)]
-pub struct Ipv6Utilization {
-    pub allocated: u128,
-    pub capacity: u128,
-}
-
 // Not really a DB model, just the result of a datastore function
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct IpPoolUtilization {
-    pub ipv4: Ipv4Utilization,
-    pub ipv6: Ipv6Utilization,
-}
-
-impl From<Ipv4Utilization> for views::Ipv4Utilization {
-    fn from(util: Ipv4Utilization) -> Self {
-        Self { allocated: util.allocated, capacity: util.capacity }
-    }
-}
-
-impl From<Ipv6Utilization> for views::Ipv6Utilization {
-    fn from(util: Ipv6Utilization) -> Self {
-        Self { allocated: util.allocated, capacity: util.capacity }
-    }
+    pub remaining: f64,
+    pub capacity: f64,
```

Comment:
I think we need to store these in the database as u128s, no? Like we *do* need integer precision here.

Reply:
This is not stored in the database